web/client/components/misc/enhancers/withFilter.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2019, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {
+    compose,
+    withState
+} from 'recompose';
+
+const withFilter = compose(
```

Comment:
This is not a global enhancer but proper of the mediaEditor, please move this in the correct directory and document a little bit. (something like `holds the state of the text filter in the media editor`)

Reply:
Why is not global? It can be applied to any filter anywhere, but ok.